components/DataSchema.tsx

```diff
@@ -78,34 +79,35 @@ const DataSchemaTable: React.FunctionComponent<{
         },
     ];
 
+    // Remove Required Column for now.f
     // add required column only if this is not a root table
-    if (!props.root) {
-        columns.push({
-            name: 'Required',
-            selector: 'required',
-            wrap: true,
-            sortable: true,
-            format: (schemaData: DataSchemaData) =>
-                schemaData.required ? 'Yes' : 'No',
-        });
-    }
+    // if (!props.root) {
+    //     columns.push({
+    //         name: 'Required',
+    //         selector: 'required',
+    //         wrap: true,
+    //         sortable: true,
+    //         format: (schemaData: DataSchemaData) =>
+    //             schemaData.required ? 'Yes' : 'No',
+    //     });
+    // }
 
     // conditionally show valid values column
     if (hasNonEmptyValidValues(props.schemaData)) {
         columns.push({
             name: 'Valid Values',
             selector: 'validValues',
             cell: (schemaData: DataSchemaData) => {
-                const text = getDataSchemaValidValues(
+                let text = getDataSchemaValidValues(
                     schemaData,
                     props.dataSchemaMap
                 )
-                    .map((s) => s.label)
-                    .join(', ');
-
+                    .map((s) => s.attribute)
+                    .join('#');
```

Comment:
it's not immediately clear here why we join with `#`. Maybe add a comment to indicate that we use `#` as a special delimiter later for formatting purposes.

Reply:
Added comments, as you suggested.  See https://github.com/ncihtan/htan-portal/pull/440/commits/496712fc830cb58a92245f74a20b54e5eb2efc89